ops/infrastructure/cljdoc.tf

```diff
@@ -65,6 +78,15 @@ resource "aws_route53_record" "dokku" {
   records  = ["167.99.133.5"]
 }
 
+resource "aws_route53_record" "test_xyz" {
```

Comment:
We probably don't need to add a record for test.cljdoc.xyz, if it's never been used before?

Reply:
That's right, I used this for testing mostly, will removed again eventually.